.github/workflows/pythonpackage.yml

```
@@ -104,9 +92,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.7
+    - name: Set up Python 3.8
       uses: actions/setup-python@v1
       with:
-        python-version: 3.7
+        python-version: 3.8
```

Save yourself the trouble of upgrading later. For flake8, there's very little chance that you won't be able to install it by the time the default Python 3 jumps up a version.

```suggestion
        python-version: 3
```